--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -584,7 +584,7 @@
       }
 
       if (!defined(use_libcxx_modules)) {
-        use_libcxx_modules = is_clang
+        use_libcxx_modules = is_clang && !(is_win && host_os == "linux")
       }
 
       if (filter_include(
@@ -715,7 +715,7 @@
       }
 
       if (!defined(use_libcxx_modules)) {
-        use_libcxx_modules = is_clang
+        use_libcxx_modules = is_clang && !(is_win && host_os == "linux")
       }
 
       if (filter_include(
