--- a/build/toolchain/gcc_toolchain.gni
+++ b/build/toolchain/gcc_toolchain.gni
@@ -369,6 +369,35 @@
       outputs = [ output ]
     }
 
+    if (current_os == "mac") {
+    tool("objc") {
+      output = "$object_subdir/{{source_name_part}}.o"
+      depfile = "$output.d"
+      precompiled_header_type = "gcc"
+
+      # {{output}} may contain .dwo file, so use $output instead of {{output}}
+      # in command.
+      command = "$coverage_wrapper$cc $md -MF $depfile {{defines}} {{include_dirs}} {{cflags}} {{cflags_objc}}${extra_cppflags}${extra_cflags}${module_name_flag} -c {{source}} -o $output"
+      depsformat = "gcc"
+      description = "OBJC {{output}}"
+      outputs = [ output ]
+    }
+
+    tool("objcxx") {
+      output = "$object_subdir/{{source_name_part}}.o"
+      depfile = "$output.d"
+      precompiled_header_type = "gcc"
+
+      # {{output}} may contain .dwo file, so use $output instead of {{output}}
+      # in command.
+      command = "$coverage_wrapper$cxx $md -MF $depfile {{defines}} {{include_dirs}} {{cflags}} {{cflags_objcc}}${extra_cppflags}${extra_cxxflags} {{module_deps}}${module_name_flag} -c {{source}} -o $output"
+      depsformat = "gcc"
+      description = "OBJCXX {{output}}"
+      outputs = [ output ]
+    }
+
+    }
+
     tool("cxx_module") {
       depfile = "{{output}}.d"
 
