diff --git a/DEPS b/DEPS
index 8ba99a6..1a5e1a5 100644
--- a/DEPS
+++ b/DEPS
@@ -769,7 +769,2 @@ deps = {
 
-  'third_party/catapult': {
-    'url': Var('chromium_git') + '/catapult.git' + '@' + Var('catapult_revision'),
-    'condition': 'not build_with_chromium',
-  },
-
   # Cherry is a dEQP/VK-GL-CTS management GUI written in Go. We use it for viewing test results.
@@ -796,7 +791,2 @@ deps = {
 
-  'third_party/dawn': {
-    'url': Var('dawn_git') + '/dawn.git' + '@' +  Var('dawn_revision'),
-    'condition': 'checkout_angle_dawn_deps and not build_with_chromium'
-  },
-
   'third_party/depot_tools': {
@@ -876,7 +866,2 @@ deps = {
 
-  'third_party/llvm/src': {
-    'url': Var('chromium_git') + '/external/github.com/llvm/llvm-project@67790d3fb06b184b7be9c4d3e36ab01f0ac4198e',
-    'condition': 'checkout_angle_cl_deps and not build_with_chromium',
-  },
-
   'third_party/llvm-build/Release+Asserts': {
@@ -1218,7 +1203,2 @@ deps = {
 
-  'third_party/SwiftShader': {
-    'url': Var('swiftshader_git') + '/SwiftShader@f3e464b1e56795940f826a972b39e19bd9f6ac6c',
-    'condition': 'not build_with_chromium',
-  },
-
   'third_party/turbine/cipd': {
@@ -1234,6 +1214,2 @@ deps = {
 
-  'third_party/VK-GL-CTS/src': {
-    'url': Var('chromium_git') + '/external/github.com/KhronosGroup/VK-GL-CTS' + '@' + Var('vk_gl_cts_revision'),
-  },
-
   'third_party/vulkan-deps': {
